packages/rum/test/viewCollection.spec.ts

```diff
@@ -108,6 +125,7 @@ describe('rum track url change', () => {
   beforeEach(() => {
     const fakeLocation: Partial<Location> = { pathname: '/foo', hash: '' }
     mockHistory(fakeLocation)
+    mockHTMLElement()
```

Comment:
since it is only used by a single test, you can probably call it or inline it there